File: c#/Selenium/Selenium/MainPage.cs
using NUnit.Framework;
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;
using OpenQA.Selenium.Firefox;
using OpenQA.Selenium.IE;
using System;

namespace Selenium
{
    class MainPage
    {
        private String url = "http://159.89.100.130/litecart/en/";
        private String expectedTitle = "My Store | Online Store";

        [Test]
        public void chromeOpen()
        {
            ChromeDriver driver = new ChromeDriver();
            driver.Navigate().GoToUrl(url);

            String actualTitle = driver.Title;
            Assert.AreEqual(expectedTitle, actualTitle);

            driver.Quit();
        }

        [Test]
        public void firefoxOpen()
        {
            IWebDriver driver = new FirefoxDriver();
            driver.Navigate().GoToUrl(url);

            String actualTitle = driver.Title;
            Assert.AreEqual(expectedTitle, actualTitle);

            driver.Quit();
        }

        [Test]
        public void explorerOpen()
        {
            IWebDriver driver = new InternetExplorerDriver();
            driver.Navigate().GoToUrl(url);

            String actualTitle = driver.Title;
            Assert.AreEqual(expectedTitle, actualTitle);

            driver.Quit();
        }
    }
}

File: java/src/test/java/MainPage.java
import io.github.bonigarcia.wdm.ChromeDriverManager;
import io.github.bonigarcia.wdm.FirefoxDriverManager;
import io.github.bonigarcia.wdm.InternetExplorerDriverManager;
import org.junit.Assert;
import org.junit.Test;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.ie.InternetExplorerDriver;

public class MainPage {

    private String url = "http://159.89.100.130/litecart/en/";
    private String expectedTitle = "My Store | Online Store";

    @Test
    public void chromeOpen(){
        ChromeDriverManager.getInstance().setup();
        WebDriver driver = new ChromeDriver();

        driver.get(url);
        String actualTitle = driver.getTitle();
        Assert.assertEquals(expectedTitle, actualTitle);
        driver.quit();
    }

    @Test
    public void firefoxOpen(){
        FirefoxDriverManager.getInstance().setup();
        WebDriver driver = new FirefoxDriver();

        driver.get(url);
        String actualTitle = driver.getTitle();
        Assert.assertEquals(expectedTitle, actualTitle);
        driver.quit();
    }

    @Test
    public void explorerOpen(){
        InternetExplorerDriverManager.getInstance().setup();
        WebDriver driver = new InternetExplorerDriver();

        driver.get(url);
        String actualTitle = driver.getTitle();
        Assert.assertEquals(expectedTitle, actualTitle);
        driver.quit();
    }
}
